core/src/main/java/org/apache/carbondata/core/indexstore/blockletindex/BlockletDataMapFactory.java

```diff
@@ -302,14 +302,18 @@ public synchronized void clear() {
     }
   }
 
-  @Override public String getCacheSize() throws IOException {
+  @Override public String getCacheSize() {
```

Comment:
Please optimize this line to two line:
`
  @Override
  public String getCacheSize() {
`

Reply:
done